subprojects/packagefiles/lcms2/meson.build

```diff
@@ -0,0 +1,74 @@
+project('Little-CMS', 'c',
+  version: '2.12'
```

Comment:
This could use `meson_version: '>=0.48.0'`

Reply:
actually 0.55 because of patch_directory.